solver/base/kustomization.yaml

```diff
@@ -1,10 +1,11 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
-resources:
-  - imagestreams.yaml
-  - openshift-templates/solve_and_sync-template.yaml
-  - argo-workflows/solver-template.yaml
 commonLabels:
   app.kubernetes.io/name: thoth
   app.kubernetes.io/component: solver
   app.kubernetes.io/managed-by: thothops-argocd
```

Comment:
this should be our ldap group name, just like in the other files.

Reply:
updated the managed-by label with our ldap group name aicoe-thoth-devops.

Reply:
+ `-argcd` because thats out team's cyborg who is doing the work ;)